src/main/java/org/broadinstitute/hellbender/utils/gcs/BucketUtils.java

```diff
@@ -393,9 +391,28 @@ public static String getPathWithoutBucket(String path) {
         final String[] split = gcsUrl.split("/");
         final String BUCKET = split[2];
         final String pathWithoutBucket = String.join("/", Arrays.copyOfRange(split, 3, split.length));
-        return CloudStorageFileSystem.forBucket(BUCKET).getPath(pathWithoutBucket);
+        CloudStorageConfiguration cloudConfig = CloudStorageConfiguration.builder()
+            // if the channel errors out, re-open up to this many times
+            .maxChannelReopens(3)
```

Comment:
Certainly! I named it NIO_MAX_REOPENS since it also controls how many times we re-open the channel (and normal retries are handled differently).